{{cookiecutter.app_name}}/test/http.go

```diff
@@ -0,0 +1,22 @@
+package test
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/onsi/ginkgo"
+)
+
+func MakeRequest(method string, path string, body io.Reader) *httptest.ResponseRecorder {
+	req, err := http.NewRequest(method, path, body)
+	if err != nil {
+		ginkgo.Fail("Failed to init request: " + err.Error())
```

Comment:
Minor preference for:

```suggestion
		ginkgo.Fail("Failed to make a request: " + err.Error())
```

Reply:
Fixed in 9f3dec0bf480c981234d62dd2d92297f1a0d0c0a